scripts/diagnose.sh

```diff
@@ -147,15 +147,19 @@ commands=(
 	'sysctl -a'
 	'systemctl list-units --failed --no-pager'
 	'top -b -n 1'
-	'grep -vE \"/var/cache/ldconfig/aux-cache|md5sum|/etc/hostname|/etc/machine-id|/etc/resin-supervisor/supervisor.conf|/etc/systemd/timesyncd.conf|/home/root/.rnd\" /resinos.fingerprint | md5sum --quiet -c ' # https://github.com/balena-os/meta-balena/issues/1618
+	'grep -vE \"/var/cache/ldconfig/aux-cache|md5sum|/etc/hostname|/etc/machine-id|/etc/balena-supervisor/supervisor.conf|/etc/resin-supervisor/supervisor.conf|/etc/systemd/timesyncd.conf|/home/root/.rnd\" /resinos.fingerprint | md5sum --quiet -c ' # https://github.com/balena-os/meta-balena/issues/1618
 
 	# SUPERVISOR specific commands
 	'echo === SUPERVISOR ==='
+	'$ENG exec balena_supervisor cat /etc/resolv.conf'
 	'$ENG exec resin_supervisor cat /etc/resolv.conf'
+	'$ENG logs balena_supervisor'
 	'$ENG logs resin_supervisor'
 	'curl --max-time 5 localhost:'"${LISTEN_PORT}"'/v1/healthy'
+	'journalctl --no-pager --no-hostname -n 200 -a -u balena-supervisor'
 	'journalctl --no-pager --no-hostname -n 200 -a -u resin-supervisor'
 	'ls -lR /tmp/*-supervisor/**/*'
+	'systemctl status balena-supervisor --no-pager'
 	'systemctl status resin-supervisor --no-pager'
```

Comment:
All this duplication is ugly but since it's just going to record the results as stdout and stderr is there any point in having a cleaner solution with resin being the fallback?
```suggestion
'/bin/bash -c "systemctl status balena-supervisor --no-pager || systemctl status resin-supervisor --no-pager"'
```

Reply:
how about 
```suggestion
	'systemctl status resin-supervisor balena-supervisor --no-pager'
```

Reply:
Looks good, I can use this on my supervisor PR as well